apps/api/entrypoint.R

```diff
@@ -0,0 +1,25 @@
+source("auth.R")
+source("ping.R")
+
+root <- plumber::plumber$new()
+root$setSerializer(plumber::serializer_unboxed_json())
+
+root$filter("require-auth", authenticate_user)
+
+root$handle("GET", "/api/ping", ping)
```

Comment:
can you document here why you use this syntax vs the plumber$new

Reply:
Added comments to the `entrypoint.R` & other relevant files